docs/media.md

```diff
@@ -0,0 +1,28 @@
+# Local Media
+
+Local Server supports the [Altis Media module](https://www.altis-dxp.com/resources/developer-docs/media/) by replicating the media handling capabilities of a deployed Altis environment. A new Local Server setup will support [Dynamic Images](https://www.altis-dxp.com/resources/developer-docs/media/dynamic-images/) and other media features, and all images you upload will be stored in a local container on your development machine.
+
+When using Local Server to work on an existing Altis site, you may need to pull down a database backup for local testing. Downloading uploaded media along with this database backup may be prohibitively time consuming, or take too much local disk space. Local Server can be configured to request these images directly from an Altis cloud environment.
+
+You can add a file `.config/nginx-additions-media.conf` to your Altis project which instructs Local Server to redirect image requests to a remote server if they were not accessible locally. This preserves your ability to upload and edit local images, while allowing any image file not present in your environment to be loaded from a remote bucket instead.
+
+Assuming that your Altis project uses the name `myproject` and you have confirmed that your environment uses the `us-east-1.tchyn.io` Tachyon URL, your `nginx-additions-media.conf` could look like this:
```

Comment:
I think that's a legacy tachyon endpoint, it can be accessed via `https://example.com/tachyon/` now so we can simplify this part of the docs a bit:
```suggestion
Assuming that your Altis project uses the name `myproject` for Local Server and your environment uses the domain name `myproject.altis.cloud`, your `nginx-additions-media.conf` could look like this:
```